benchmarks/nemo/nemolite2d/manual_versions/psykal_kokkos/time_step_kokkos.cpp

```diff
@@ -84,10 +84,10 @@ extern "C" void c_invoke_time_step(
         }
     );
 
-    // Momentum_u kernel (internal domain)
+    // Momentum_u kernel (internal domain but top x)
```

Comment:
Apologies if I'm being picky but I don't understand what 'but top x' means. Perhaps you want "internal u points" and similar for v below?

Reply:
Yes, that sounds better, changed.